build-tools/automation/azure-pipelines.yaml

```diff
@@ -113,6 +113,18 @@ stages:
       parameters:
         condition: and(succeededOrFailed(), eq(variables['MicroBuildSignType'], 'Real'))
 
+    - script: >
+        mkdir -p $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/windows-toolchain-pdb &&
+        ln $(System.DefaultWorkingDirectory)/xamarin-android/bin/$(XA.Build.Configuration)/lib/xamarin.android/xbuild/Xamarin/Android/binutils/bin/*.pdb .
+      workingDirectory: $(System.DefaultWorkingDirectory)/xamarin-android
+      displayName: copy Windows toolchain pdb files
+
+    - task: PublishPipelineArtifact@1
+      displayName: upload Windows toolchain pdb files
+      inputs:
+        artifactName: $(WindowsToolchainPdbArtifactName)
+        targetPath: $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/windows-toolchain-pdb
+
```

Comment:
It looks like this isn't picking up any files for some reason: 
> Processed 0 files from /Users/builder/azdo/_work/2/s/xamarin-android/bin/BuildRelease/windows-toolchain-pdb successfully.

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6021318&view=logs&j=96fd57f5-f69e-53c7-3d47-f67e6cf9b93e&t=ec7023ec-6baf-5148-a0bd-5c44ba115358&l=15

Reply:
The `ln` target path was set to `.` (e.g. $(System.DefaultWorkingDirectory)/xamarin-android) instead of $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/windows-toolchain-pd